services-camel/src/test/resources/triples-reindexing-it-config.properties

```diff
@@ -0,0 +1,28 @@
+fcrepo.baseUrl=http://localhost:8080/fcrepo/rest
+fcrepo.baseUri=localhost:8080/fcrepo/rest
+fcrepo.serverUri=http://localhost:8080/
```

Comment:
i wish we could get more consistent about how we name URLs and URIs throughout our codebase. seems like this one should be serverUrl to match baseUrl

Reply:
Yeah, there isn't really any clear distinction, I think we will have to clean them up everywhere at once in another ticket